lib/application.js

```diff
@@ -5,6 +5,7 @@ var DevNull = require('dev-null')
 var fs = require('fs')
 var path = require('path')
 var WebDriver = require('webdriverio')
+var checkMinorVersion = require('./check-minor-version')
```

Comment:
Super minor, but these requires were previously sorted, would be nice to preserve.